src/main/java/uk/gov/hmcts/reform/sscs/service/CaseFlagMappingService.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.helper.FlagCode;
+import uk.gov.hmcts.reform.sscs.model.DescendantCaseData;
+import uk.gov.hmcts.reform.sscs.model.HearingWrapper;
+
+import java.util.Map;
+import javax.validation.constraints.NotNull;
+
+@Service
```

Comment:
I don't think this should be a service, it should be a static utility class with no state